WooCommerce/Classes/POS/Presentation/PointOfSalePaymentsTestView.swift

```diff
@@ -0,0 +1,56 @@
+import SwiftUI
+import Combine
+
+// Required for making a dummy order
+import Yosemite
+
+class PointOfSalePaymentsTestViewModel: ObservableObject {
+    let cardPresentPayments: CardPresentPayments
+    var cancellables = Set<AnyCancellable>()
+
+    @Published var onboardingViewModels: InPersonPaymentsViewModel?
```

Comment:
nit: just to match the value type
```suggestion
    @Published var onboardingViewModel: InPersonPaymentsViewModel?
```